Dockerfile

```
@@ -0,0 +1,15 @@
+FROM node:18-alpine  
+
+WORKDIR /app
+
+COPY package*.json ./  
+
+RUN npm ci
+
+COPY . .
```

Why are we doing a copy twice? Will it work if you do a single copy? eg.
```suggestion
COPY . .

RUN npm ci
```